rustls/src/conn.rs

```diff
@@ -811,13 +811,15 @@ impl<Data> From<ConnectionCore<Data>> for ConnectionCommon<Data> {
 pub struct UnbufferedConnectionCommon<Data> {
     pub(crate) core: ConnectionCore<Data>,
     wants_write: bool,
+    emitted_peer_closed: bool,
```

Comment:
Would this be a touch clearer named `emitted_peer_closed_state` to disambiguate with the actual alert on the wire or am I over-thinking?